Find a functionally equivalent code:android.content.RestrictionEntry.setAllSelectedStrings:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the current list of selected values for an entry of type {@link #TYPE_MULTI_SELECT}.
 * These values will be persisted by the system for later use by the application.
 * @param allSelectedStrings the current list of selected values.
 */

@@ -1,3 +1,3 @@
 {
-    currentValues = allSelectedStrings;
+    mCurrentValues = allSelectedStrings;
 }

